docs/Policies/App Policies/index.md

```diff
@@ -0,0 +1,247 @@
+    "123abc": {
+        "keep_context": false,
+        "steal_focus": false,
+        "priority": "NONE",
+        "default_hmi": "NONE",
+        "groups": ["Base-4"],
+        "RequestType": [],
+        "RequestSubType": [],
+        "nicknames": ["Example Application"],
+        "AppHMIType": ["NAVIGATION", "MEDIA"]
+    },
+    ...
+}
+```
+
+## Security Fields
+
+### certificate
+
+A string field containing a certificate for establishing a secure connection with the application. Currently only used for Cloud Applications when establishing a secure websocket connection.
+
+### encryption_required
+
+This flag which can be used to override `encryption_required` on the functional group level. If `false`, then this application will not be required to encrypt any of the RPCs which it has permissions to, even if the associated functional group has `encryption_required` set to `true`. Defaults to `true` if omitted.
+
+### Example Entry
+
+```json
+"app_policies": {
+    "123abc": {
+        "keep_context": false,
+        "steal_focus": false,
+        "priority": "NONE",
+        "default_hmi": "NONE",
+        "groups": ["Base-4"],
+        "RequestType": [],
+        "RequestSubType": [],
+        "certificate": "-----BEGIN CERTIFICATE-----\n...",
+        "encryption_required": true
+    },
+    ...
+}
+```
+
+## Remote Control Fields
+
+!!! note
+These permissions will only take effect if the application registers with the `REMOTE_CONTROL` AppHMIType
+!!!
+
+### moduleType
+
+Defines a list of Remote Control module types (defined in the `ModuleType` enum in the [Mobile API](https://github.com/smartdevicelink/rpc_spec#moduletype)) which are accessible by the app. If empty, all module types will be allowed. If omitted, no module types will be allowed.
+
+### Example Entry
+
+```json
+"app_policies": {
+    "123abc": {
+        "keep_context": false,
+        "steal_focus": false,
+        "priority": "NONE",
+        "default_hmi": "NONE",
+        "groups": ["Base-4", "RemoteControl"],
+        "RequestType": [],
+        "RequestSubType": [],
+        "moduleType": ["RADIO", "CLIMATE"]
+    },
+    ...
+}
+```
+
+## App Service Fields
+
+These fields are used if the application is either be a provider or consumer of app service data. See our [App Service Provider Guide](https://smartdevicelink.com/en/guides/core/app-service-provider-guide/) for more information on developing an application with these capabilities.
+
+### app_services
+
+This field defines the properties of the app services that this application is allowed to publish. Each key within this object corresponds to an app service type (ex. "MEDIA").
+
+#### service_names
+
+Defines a list of potential serviceNames that an app can publish with this service type. If the serviceName field in the app's `PublishAppService.appServiceManifest` does not match one of the values in this list, the message will be rejected. If omitted, no restrictions will be applied to serviceName.
```

Comment:
```suggestion
Defines a list of potential service names that an app can publish with this service type. If the `serviceName` field in the app's `PublishAppService.appServiceManifest` does not match one of the values in this list, the message will be rejected. If omitted, no restrictions will be applied to the service name.
```

I feel like it should be 'service names' in the first line, maybe in backticks the second time, and then with the space again the last time.